analytics.js

```diff
@@ -926,11 +926,12 @@ class Analytics {
       const tempClientSuppliedIntegrations = Object.keys(
         clientSuppliedIntegrations
       )
-        .filter(
-          (integration) =>
-            integration !== "All" &&
-            clientSuppliedIntegrations[integration] !== true
-        )
+        .filter((integration) => {
+          if (integration !== "All") {
+            return clientSuppliedIntegrations[integration] !== true;
+          }
+          return integration;
```

Comment:
This should be `true`, right?

Reply:
Since All key value will always set to true, it will always return true

Reply:
I mean, `integration` is the name of the integration, which is a string. We should be returning `true` instead to avoid filtering it. It should still work as all non-empty strings evaluate to truthy values but let's still fix it.

Reply:
updated